py3-packaging.yaml

```diff
@@ -1,23 +1,31 @@
 package:
   name: py3-packaging
   version: "24.1"
-  epoch: 0
+  epoch: 1
   description: "core utilities for python3 packaging"
   copyright:
     - license: Apache-2.0 AND BSD-2-Clause
   dependencies:
-    runtime:
-      - py3-parsing
-      - python3
+    provider-priority: 0
```

Comment:
What does this mean? In Melange it says it will determines the priority of provides against other equal package provides

Reply:
This is part of the multi versioned python support, which allows each versioned py3.MN-foo to provide py3-foo and default to the highest priority 3.12 - this probably needs documentation.